README.md

```diff
@@ -36,21 +36,22 @@ dependencies:
 flutter packages get
 ```
 
-### Using Instabug
+### Import Instabug
```

Comment:
I personally prefer the former title. However, we could reorder the steps so that "Initializing" comes before "Using", what do you think?

Reply:
I separated the steps into only Importing and Initializing. `Using` is a bit misleading since it can point to `using` APIs for example.

We cant re-order the steps since the Initializing will not work unless you import instabug

Reply:
I see. Given that importing is a crucial step in initializing, do we really need to separate them?